### PR TITLE
Simplifies crimes + hud clean up

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Ported PRs: https://github.com/tgstation/tgstation/pull/45476
https://github.com/tgstation/tgstation/pull/49591

I removed minor and major crimes, reduced amount of pop ups you need to fill while adding a crime, added an option to add the details to a crime (only if it has no details).
Huds will now add TRAIT_something_HUD to a mob.

## Why It's Good For The Game

HUD traits weren't even used before.

## Changelog
:cl:
del: Major and minor crimes are gone.
tweak: Adding a crime recond in sechuds in only 1 pop-up now.
add: You can add details to a crime separately.
refactor: Cleaned up sec console code and hud glasses code.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -280,244 +280,250 @@
...
+		if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD) && !HAS_TRAIT(H, TRAIT_MEDICAL_HUD))
+			return
+		var/datum/data/record/R = find_record("name", perpname, GLOB.data_core.general)
+		if(href_list["photo_front"] || href_list["photo_side"])
+			if(!R)
+				return
+			if(!H.canUseHUD())
+				return
+			if(!HAS_TRAIT(H, TRAIT_SECURITY_HUD) && !HAS_TRAIT(H, TRAIT_MEDICAL_HUD))
+				return
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/human.dm]: Redundant check due to line 287

File: code/modules/surgery/organs/augments_eyes.dm [hunk 2]
```diff
@@ -13,28 +13,35 @@
 	desc = "These cybernetic eyes will display a HUD over everything you see. Maybe."
 	slot = ORGAN_SLOT_HUD
 	var/HUD_type = 0
+	var/HUD_trait = null
```
Comment (PowerfulBacon) [inline code/modules/surgery/organs/augments_eyes.dm]:
```suggestion
	var/HUD_trait
```

File: code/modules/surgery/organs/augments_eyes.dm [hunk 3]
```diff
@@ -13,28 +13,35 @@
 	desc = "These cybernetic eyes will display a HUD over everything you see. Maybe."
 	slot = ORGAN_SLOT_HUD
 	var/HUD_type = 0
```
Comment (PowerfulBacon) [inline code/modules/surgery/organs/augments_eyes.dm]:
```suggestion
	var/HUD_type
```

**Comment by Isy232** [comment]
This sounds cool, sechuds were a fucking pain to add crimes with

**Review by qwertyquerty — APPROVED** [review]

**Comment by PowerfulBacon** [comment]
Looks good

**Pull request merged**